wordpress-mobile/WordPress-Android
# Insights redesign

This PR introduces a new block containing 2 columns with label and value and uses this block in Today stats, All time stats and Most popular stats. It's purpose is to allow the user to quickly scan the insights.

The additional information that's now hidden (the best date and the % of most popular day/hour) are now shown on click in a tooltip.

To test:
* Go to Stats/Insights
* Check the Today's stats, All-time stats and Most popular stats 
* The data has not changed

![Screenshot_1553087213](https://user-images.githubusercontent.com/1079756/54688197-7a547180-4b1d-11e9-9758-4f8dbdbd3f02.png)

Update release notes:
